Makefile

```diff
@@ -18,3 +18,15 @@ ifndef VIRTUAL_ENV
 else
 	$(warning VIRTUAL_ENV variable present, already within a virtual environment?)
 endif
+
+cookie-teardown:
```

Comment:
I think this requires the `venv_check` as a prerequisite to ensure we are on a virtual environment (so that `deactivate` is present in the environment)

Reply:
Good point! I added the check now